packages/auth-nextjs/src/shared.ts

```diff
@@ -65,8 +74,12 @@ export class NextAuthSession {
 
   async isLoggedIn() {
     if (!this.authToken) return false;
-    return (await this.client.querySingle(
-      `select exists global ext::auth::ClientTokenIdentity`
-    )) as boolean;
+    try {
+      return (await this.client.querySingle(
+        `select exists global ext::auth::ClientTokenIdentity`
+      )) as boolean;
```

Comment:
I know this isn't new and is definitely not a blocker, but can we move the `boolean` into the generic position rather than using a cast here?

```suggestion
      return await this.client.querySingle<boolean>(
        `select exists global ext::auth::ClientTokenIdentity`
      );
```